addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -449,58 +449,11 @@ weSnippetEditor.SnippetEditor.include({
 // Edit mode customizations of public widgets.
 
 publicWidget.registry.hoverableDropdown.include({
-    /**
-     * @override
-     */
-    start() {
-        if (this.editableMode) {
-            this._onPageClick = this._onPageClick.bind(this);
-            this.el.closest('#wrapwrap').addEventListener('click', this._onPageClick, {capture: true});
-        }
-        return this._super.apply(this, arguments);
-    },
-    /**
-     * @override
-     */
-    destroy() {
-        if (this.editableMode) {
-            this.el.closest('#wrapwrap').removeEventListener('click', this._onPageClick, {capture: true});
-        }
-        return this._super.apply(this, arguments);
-    },
-
-    //--------------------------------------------------------------------------
-    // Private
-    //--------------------------------------------------------------------------
-    
-    /**
-     * Hides all opened dropdowns.
-     *
-     * @private
-     */
-    _hideDropdowns() {
-        for (const toggleEl of this.el.querySelectorAll('.dropdown.show .dropdown-toggle')) {
-            $(toggleEl).dropdown('hide');
-        }
-    },
```

Comment:
As this is for stable, you should keep this method and add a `// TODO remove in master` (and make sure to remove it in the master forward-port, obviously)